branch_training/vlado's brand new poem

```diff
@@ -0,0 +1,3 @@
+This poem starts here
+and ends in the next line
+.
```

Comment:
come on e vladimir, you can do better. a haiku has 3 lines.

Reply:
added new lines.